src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 public interface MachineProducer {
```

Comment:
should be parametrized too

Reply:
Here should be a type parameter. This parameter will be equal to Bulldozer for BulldozerProducer, Truck for TruckProducer, etc. Then you reuse that parameter to return a List of exactly these Class instances in `get` method.